rust-connectors/common/src/config.rs

```diff
@@ -68,6 +71,19 @@ pub struct ProducerParameters {
     batch_size: Option<ByteSize>,
 }
 
+#[allow(clippy::derive_partial_eq_without_eq)]
+#[derive(Debug, Clone, PartialEq, Deserialize, Serialize)]
+pub struct TransformParameters(pub Vec<TransformStep>);
```

Comment:
Hmm. Is there a particular reason for using a wrapper struct here rather than just having `pub transforms: Vec<TransformStep>` in the `ConnectorConfig` struct? If it just felt _cleaner_, I'm fine with that.

Reply:
You're right. Removed that wrapper.